### ISSUE TITLE
Talkback cannot get to content inside Gherkin or WCAG checkbox region

### ISSUE THREAD

**Issue by silverli** [tmobile/magentaA11y#52]
Talkback on Google Pixel 4a 5G skips over the content inside the success criteria region completely.

see recording below

https://github.com/tmobile/magentaA11y/assets/12848168/c4c63cfb-f35e-400b-ae42-11b844095af1

**Comment by charlietriplett** [comment]
This issue varies by device/operating system version. Unable to replicate but will be mindful of this.

**Comment by silverli** [comment]
I have been able to replicate this issue (not being able to get Talkback into the Gherkin or WCAG checkbox region for any web component) on:

* Google Pixel 4a 5g
* Google Pixel 7 (@jzoo tested for me)
* Samsung Galaxy s21 ultra 5g
* Revvl6 5g

Would like to reopen and I will investigate this weirdness!
